pyproject.toml

```diff
@@ -59,10 +59,10 @@ pyreadline3 = {version = "^3.4.1", python = "^3.5"}
 hidapi = "^0.12.0.post2"
 rich = {version = "^13.2.0", optional = true}
 requests = {version = "^2.28.2", optional = true}
-
+isort = ">=5.11.4"
+black = "22.10.0"
```

Comment:
Why?

Reply:
The pipeline with tox were having import issue and so the tests were not runned.
https://ci.eclipse.org/kiso-testing/job/kiso-testing/job/PR-244/3/console

The solution I found were changing the dependencies since pykiso to pytest need these dependencies : 
https://github.com/eclipse/kiso-testing/blob/master/src/pykiso/tool/pykiso_to_pytest/cli.py#L17-L18

Reply:
If you have to add these deps, create an extra called e.g. `pytest`

Reply:
Done